src/view/DashboardView.java

```diff
@@ -100,6 +108,22 @@ public void actionPerformed(ActionEvent e) {
         });
         this.setLayout(new GridLayout());
         this.add(main);
+
+        deleteButton.addActionListener(new ActionListener() {
```

Comment:
From my understanding, this code allows the user to select multiple rows in the table and delete multiple at a time. However, this is an unintended feature since we want the user to only be able to select one row at a time, so as to be consistent with other use cases which require this for their functionality (i.e. update account requires that you can only select one row so that it knows which one to update).

This may have implications for other parts of the use case chain, such as the controller's execute method and so on

Reply:
This didn't have any further implications. In my most recent commit, I fixed this and implemented it in the way you outlined by doing what the Update Account use case does: just updating the account furthest to the top of the table, and giving you a JOptionPane with a small message if you don't have any accounts selected.